elftools/elf/elffile.py

```diff
@@ -69,7 +70,7 @@ class ELFFile(object):
             e_ident_raw:
                 the raw e_ident field of the header
     """
-    def __init__(self, stream):
+    def __init__(self, stream, stream_loader=None):
```

Comment:
The docstring has to explain this new parameter in detail (from the POV of its usage)